scripts/globals/instance.lua

```diff
@@ -1,64 +1,69 @@
+-----------------------------------
+-- Instance Helpers
+-----------------------------------
+local instance_helpers = {}
 
-function updateInstanceTime(instance, elapsed, texttable)
-    local players = instance:getChars()
+local function setInstanceLastTimeUpdateMessage (instance, players, remainingTimeLimit)
+    local message = 0
     local lastTimeUpdate = instance:getLastTimeUpdate()
+
+    if lastTimeUpdate == 0 and remainingTimeLimit < 600 then
+        message = 600
+    elseif lastTimeUpdate == 600 and remainingTimeLimit < 300 then
+        message = 300
+    elseif lastTimeUpdate == 300 and remainingTimeLimit < 60 then
+        message = 60
+    elseif lastTimeUpdate == 60 and remainingTimeLimit < 30 then
+        message = 30
+    elseif lastTimeUpdate == 30 and remainingTimeLimit < 10 then
+        message = 10
+    end
+
+    if message ~= 0 then
+        for i, player in pairs(players) do
+            if (remainingTimeLimit >= 60) then
```

Comment:
Could you please remove the containing parentheses for this conditional to match style guide?

Reply:
Nice catch ! it's fix!